x-pack/plugins/security_solution/server/lib/timeline/routes/notes/delete_note.ts

```diff
@@ -46,15 +46,26 @@ export const deleteNoteRoute = (
         try {
           const frameworkRequest = await buildFrameworkRequest(context, security, request);
           const noteId = request.body?.noteId ?? '';
+          const noteIds = request.body?.noteIds ?? null;
+          if (noteIds != null) {
+            const res = await deleteNote({
+              request: frameworkRequest,
+              noteIds,
+            });
 
-          const res = await deleteNote({
-            request: frameworkRequest,
-            noteId,
-          });
+            return response.ok({
+              body: { data: { persistNote: res } },
+            });
+          } else {
```

Comment:
```suggestion
          } else if (noteId !== '') {
```

nit, but you can probably skip this if the noteId wasn't provided